### DISCUSSION TITLE
Sessions Rating Platform

### DISCUSSION THREAD

**Discussion by way0utwest** [dataplat/DataSaturdays#163] [Ideas]
In what way do we want to allow speaker and session rating?
- build platform
- integrate with existing platform

In addition, what features are needed:
- rate session/speaker/event in standard questions
- allow event customization of questions for session/speaker/event
- allow organizers to enter ratings (i.e. attendees vote with cards as they leave a room)
- reports for speakers
- reports for organizers
- aggregation of reports for sessions/speakers/events over time (multiple events)
- redaction of inappropriate comments
- public view of ratings (anonymous allowed?)

**Comment by HugoKornelis** [comment]
One possible option would be to use speakerscore.com. They can also host the slide deck and democode, and an attendee would first get asked to evaluate a session before gaining access to the resources. (They can skip the evaluation, but it does offer a good incentive).
Down side for the event organizers is that their feedback questions are presenter focused. Event organizers often want to get feedback on the overall schedule, the venue, etc etc.

Note: Last time I used speakerscore.com is a few years back, they might have changed things since.
